jest.setup.ts

```diff
@@ -2,4 +2,8 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import "@testing-library/jest-dom";
+
+beforeEach(() => {
+  expect.hasAssertions();
+});
```

Comment:
Make sure a test which doesn't remember to actually test anything won't incorrectly show as passing 🙃